cmd/di.go

```diff
@@ -610,9 +610,8 @@ func (di *Dependencies) bootstrapQualityComponents(options node.OptionsQuality)
 }
 
 func (di *Dependencies) bootstrapLocationComponents(options node.OptionsLocation, configDirectory string) (err error) {
-	_, err = firewall.AllowURLAccess(options.IPDetectorURL)
-	if err != nil {
-		return err
+	if _, err = firewall.AllowURLAccess(options.IPDetectorURL); err != nil {
+		return
```

Comment:
I thought we agreed to avoid naked returns. It's bad for readability. Now I have to go back to the function signature to check what's needs to be returned and then scan all the lines to determine what's actually being returned.